src/tools/web/clusterd/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "clusterd",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "npm run build && npm run start:prod",
+    "start:verbose": "npm run build && npm run start:prod:verbose",
+    "build": "babel -d build/ src/",
+    "start:prod": "cross-env DEBUG=clusterd:*,-*debug,-*log node build/",
+    "start:prod:verbose": "cross-env DEBUG=clusterd:* node build/",
+    "start:dev": "cross-env NODE_ENV=development DEBUG=clusterd:* babel-watch src/",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/omnidan/elektra-web.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/omnidan/elektra-web/issues"
+  },
+  "homepage": "https://github.com/omnidan/elektra-web#readme",
+  "dependencies": {
+    "body-parser": "^1.15.2",
+    "cors": "^2.8.1",
```

Comment:
in that case clusterd would also have to serve the client - and then I still need to figure out a way to do that while keeping the projects in separate folders (it would get a little messy if everything was in the clusterd folder), but it should be possible